src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -81,21 +63,29 @@ const QuizzesContainer = () => {
   )
 
   const deleteQuiz = useCallback(
-    (id?: string) => ResourceService.deleteQuiz(id ?? ''),
-    []
+    async (id?: string): Promise<AxiosResponse<unknown>> => {
+      const response = await ResourceService.deleteQuiz(id ?? '')
+      await queryClient.invalidateQueries({ queryKey: ['quizzes'] })
+      return response
+    },
+    [queryClient]
   )
 
-  const { response, loading, fetchData } = useAxios<
-    ItemsWithCount<Quiz>,
-    GetResourcesParams
-  >({
-    service: getQuizzes,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError
+  const {
+    data: quizzes,
+    isLoading,
+    error,
+    refetch
+  } = useQuery({
+    queryKey: ['quizzes', itemsPerPage, sort, page, selectedItems],
+    queryFn: getQuizzes,
+    options: {
+      staleTime: Infinity
+    }
   })
 
   const onEdit = (id: string) => {
-    const resource = response.items.find((item) => item._id === id)
+    const resource = quizzes?.items.find((item) => item._id === id)
```

Comment:
Let's add exit condition here
```suggestion
    if(!quizzes){
      return
    }
    
    const resource = quizzes.items.find((item) => item._id === id)
```

And also please try to get rid of `createUrlPath` inside this component. Replace it with `getFullUrl` and if you will face some problems with it - message me